test/e2e/framework/util.go

```diff
@@ -675,18 +675,18 @@ func ContainerInitInvariant(older, newer runtime.Object) error {
 	if err := initContainersInvariants(newPod); err != nil {
 		return err
 	}
-	oldInit, _, _ := e2epod.Initialized(oldPod)
-	newInit, _, _ := e2epod.Initialized(newPod)
+	oldInit, _, _ := initialized(oldPod)
+	newInit, _, _ := initialized(newPod)
 	if oldInit && !newInit {
-		// TODO: we may in the future enable resetting Initialized = false if the kubelet needs to restart it
+		// TODO: we may in the future enable resetting initialized = false if the kubelet needs to restart it
 		// from scratch
 		return fmt.Errorf("pod cannot be initialized and then regress to not being initialized")
 	}
 	return nil
 }
 
 func initContainersInvariants(pod *v1.Pod) error {
-	allInit, initFailed, err := e2epod.Initialized(pod)
+	allInit, initFailed, err := initialized(pod)
```

Comment:
[initContainersInvariants](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/util.go#L688) should also be refactored along with `ContainerInitInvariants()` as it is only used by that function.